sdk_container.sh

```diff
@@ -31,6 +31,20 @@ export CC=gcc-7
 export CXX=g++-7
 export CPP=cpp-7
 
+# goland
+
+wget https://golang.org/dl/go1.16.6.linux-amd64.tar.gz 
+rm -rf /usr/local/go && tar -C /usr/local -xzf go1.16.6.linux-amd64.tar.gz
+rm -rf go1.16.6.linux-amd64.tar.gz
+export PATH=$PATH:/usr/local/go/bin
```

Comment:
I don't think you need Go anymore? I thought it was added as a build dependency for Bazelisk, but we're not building it anymore.